src/azure-cli/azure/cli/command_modules/acs/agentpool_decorator.py

```diff
@@ -1601,6 +1615,14 @@ def add_agentpool(self, agentpool: AgentPool) -> AgentPool:
             headers=self.context.get_aks_custom_headers(),
         )
 
+    def set_up_agentpool_network_profile(self, agentpool: AgentPool) -> AgentPool:
+        self._ensure_agentpool(agentpool)
+        ip_tags = self.context.get_ip_tags()
+        if ip_tags:
+            agentpool.network_profile = self.models.AgentPoolNetworkProfile()
```

Comment:
this might remove existing configs

Reply:
yes, but I need https://github.com/Azure/azure-cli/pull/27900 to get merged first before I can fix this.

Reply:
fixed